test/test_ops.py

```diff
@@ -1424,21 +1427,49 @@ def fn(input, weight, bias):
                 cotangents = torch.randn_like(result, device=device)
                 self._compare_jacobians_of_vjp(fn, (cotangents, input, weight, bias))
 
-    @ops(filter(lambda op: op.name == "nn.functional.group_norm", functorch_lagging_op_db + additional_op_db),
-         allowed_dtypes=(torch.float32, torch.double))  # TODO: generalize
-    def test_group_norm_backward(self, device, dtype, op):
-        # hacky, only works since no group norm inputs can be scalars
-        def was_skipped_from_batched_tensors(batched_out, batch_size):
-            return batched_out.shape == (batch_size,) and all(tuple(e == 1 for e in batched_out))
+    @skipOps('TestOperators', 'test_vmap_autograd_grad', {
+        # randomness errors
+        xfail('bernoulli'),
+        xfail('nn.functional.dropout2d'),
+        xfail('nn.functional.dropout'),
+        xfail('nn.functional.feature_alpha_dropout', 'with_train'),
+        xfail('nn.functional.fractional_max_pool2d'),
+        xfail('nn.functional.fractional_max_pool3d'),
+        xfail('nn.functional.rrelu'),
+        xfail('normal'),
+        xfail('normal', 'number_mean'),
+        xfail('pca_lowrank'),
+        xfail('svd_lowrank'),
+
+        # call inplace functions
+        xfail('linalg.eigh'),  # inplace
+        xfail('linalg.householder_product'),  # inplace
+        xfail('matrix_exp'),  # inplace
+        xfail('symeig'),  # inplace
+        xfail('take'),  # inplace
+
+        xfail('linalg.eig'),  # all close?
+        # The size of tensor a (4) must match the size of tensor b (10) at non-singleton dimension 0
+        xfail('masked_select'),
+        xfail('nn.functional.max_unpool2d', 'grad'),  # contiguous call
+        xfail('nn.functional.max_unpool2d'),  # contiguous call
+        xfail('to_sparse'),  # dispatch key issue
 
-        sample_inputs = op.sample_inputs(device, dtype, requires_grad=True)
+        # numerical inconsistencies, look like bugs
+        xfail('nn.functional.binary_cross_entropy_with_logits', dtypes=(torch.float32,)),
+        xfail('ldexp', dtypes=(torch.float32,)),
+    })
+    @ops(functorch_lagging_op_db + additional_op_db, allowed_dtypes=(torch.float32, torch.double))
+    def test_vmap_autograd_grad(self, device, dtype, op):
+        if not op.supports_autograd:
+            self.skipTest("Skipped! Autograd not supported.")
+            return
 
+        sample_inputs = op.sample_inputs(device, dtype, requires_grad=True)
         for sample_input in sample_inputs:
             cotangents = get_sample_cotangents(op, sample_input)
             f, args = get_autograd_fn_and_args_with_cotangents(op, sample_input, cotangents)
             for loop_out, batched_out in get_fallback_and_vmap_exhaustive(f, args, {}, opinfo=op):
```

Comment:
Actually, I'm a little confused. Why isn't this just:
```
output = op(*sample_input.args, **sample_input.kwargs)
cotangents = tree_map(torch.randn_like, output)
def compute_grad(cotangents):
  return torch.autograd.grad(get_differentiable(output), get_differentiable(sample_inputs), cotangents, retain_grad=True)
 
for loop_out, batched_out in get_fallback_and_vmap_exhaustive(compute_grad, cotangents, {}, opinfo=op):
  self.assertEqual(loop_out, batched_out)
```
This gives us a couple of benefits:
1. avoids re-computing the forward pass multiple times
2. I'm actually not sure what get_autograd_fn_and_args_with_cotangents is doing, but is this similar?